DevOps/pipeline/azure-pipelines-gw.yml

```diff
@@ -5,7 +5,7 @@ resources:
       type: github
       endpoint: DfE-Digital
       name: DFE-Digital/login.dfe.devops
-      ref: main
+      ref: feature/DSI-6157
```

Comment:
feature branch?

Reply:
Thanks I was going to updated before and slip again